### DISCUSSION TITLE
Inconsistency between the CCO Modeling Information Doc and Ratio ICE definition?

### DISCUSSION THREAD

**Discussion by PhiBabs935** [CommonCoreOntology/CommonCoreOntologies#408] [Announcements]
In "Modeling Information with the Common Core Ontologies 1.3", specifically in section 4.10 (Car Modeling Specification example), an automobile is said to have a Range Function, and this function is said to be measured by a Ratio Measurement ICE. On the other hand, R-MICE has the definition: 'A Measurement Information Content Entity that consists of a symbol that places a Quality of an Entity onto an interval scale having a true zero value.' 

Is the example in the doc technically not correct? (The range measurement instead being a measure of a quality of the hypothetical path the vehicle traverses before maxing out its charge?)

Or can R-MICEs be extended to functions? That incidentally would be great for me, as it would simplify a modeling case I am tackling at the moment (range capability of a sensor).

**Comment by cameronmore** [comment]
This makes me wonder what 'range' is entirely, whether it's the function of something. Is the function of a sensor to sense? And the range is just a disposition? Unless the sensor is built specifically to sense range, I wonder whether range is a function. If range is a disposition, which inheres in sensors that have sensing functions, then we definitely want to say we can measure or track information about the sensor. I'm happy to hear others weigh in @APCox @mark-jensen

**Comment by mark-jensen** [comment]
@PhiBabs935 Yes, the modeling doc is wrong. Probably result of attempting to simplify the complexity of artifact specifications. 

The def for 'measurement ICE' is "A Descriptive Information Content Entity that describes the extent, dimensions, quantity, or quality of an Entity relative to some standard."

This makes me wonder if that def excludes measuring dispositions (realizables in generally probably), which we do, esp. with 'nominal measurement ICE'. It's a good catch and needs to discussed further. Clearly we can measure the realization of functions, but "extent, dimensions, quantity, or quality" does not seem to apply to the function itself.

**Comment by neilotte** [comment]
There are multiple issues related to updating documentation that are presently being assessed on the discussion board. I will move this item over there so that it may be tracked.
